conda/recipes/cugraph-nx/meta.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set py_version = environ['CONDA_PY'] %}
+{% set date_string = environ['RAPIDS_DATE_STRING'] %}
+
+package:
+  name: cugraph-nx
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  build:
+      number: {{ GIT_DESCRIBE_NUMBER }}
+      string: py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+
+requirements:
+  host:
+    - python
+  run:
+    - pylibcugraph ={{ version }}
+    - networkx >=3.0
+    - cupy >=12.0.0
+    - python
+
+tests:
+  imports:
+    - cugraph_nx
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://rapids.ai/
+  license: Apache-2.0
+  license_file: ../../../LICENSE
+  summary: cuGraph backend for GPU-accelerated NetworkX
```

Comment:
Why don't any of the conda recipes have
```yaml
  dev_url: https://github.com/rapidsai/cugraph
```
?

Reply:
Good question!  I added those to all recipes in the most recent commit.